playbooks/roles/caasp4/templates/skuba-terraform-openstack.tfvars.j2

```diff
@@ -29,7 +29,7 @@ packages = [
   "kernel-default",
   "-kernel-default-base",
   "ca-certificates-suse",
-  "patterns-caasp-Node",
+  "patterns-caasp-Node-1.15",
```

Comment:
You can remove this, skuba takes care of it.

Reply:
I already asked this and the answer from @Itxaka is here: https://github.com/SUSE-Cloud/socok8s/pull/657#discussion_r310976875